src/Chassis.Fairing.Mvc/ChassisMvcBootstrapper.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Web.Mvc;
+using Autofac.Integration.Mvc;
+using Chassis.Apps;
+
+namespace Chassis.Fairing.Mvc
+{
+    public static class ChassisMvcBootstrapper
+    {
+        public static void Bootstrap(IApplication app)
+        {
+            DependencyResolver.SetResolver(new AutofacDependencyResolver(app.Container));

```

Comment:
this is currently called after startup in fulfillment, and after all routes, so yes - this can move into a startup function. :+1: